tutorials/deploy-local/1.deploy-local.ipynb

```diff
@@ -0,0 +1,431 @@
+    "\n",
+    "The options available for a deployment configuration differ depending on the compute target you choose. In a local deployment, all you can specify is which port your webservice will be served on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9762e02b",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "service = Model.deploy(ws, \"myservice\", [model], inference_config, deployment_config)\n",
+    "service.wait_for_deployment(show_output=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ede851f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(service.get_logs())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b9d47b4",
+   "metadata": {},
+   "source": [
+    "## Call into your model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45e92e91",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import json\n",
+    "\n",
+    "uri = service.scoring_uri\n",
+    "requests.get(\"http://localhost:6789\")\n",
+    "headers = {\"Content-Type\": \"application/json\"}\n",
+    "data = {\n",
+    "    \"query\": \"What color is the fox\",\n",
+    "    \"context\": \"The quick brown fox jumped over the lazy dog.\",\n",
+    "}\n",
+    "data = json.dumps(data)\n",
+    "response = requests.post(uri, data=data, headers=headers)\n",
+    "print(response.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34c9e82c",
+   "metadata": {},
+   "source": [
+    "Notice the use of the AZUREML_MODEL_DIR environment variable to locate your registered model. Now that you've added some pip packages, you also need to update your inference configuration to add in those additional packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71b9d8a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "env = Environment(name=\"myenv\")\n",
+    "python_packages = [\"nltk\", \"numpy\", \"onnxruntime\"]\n",
+    "for package in python_packages:\n",
+    "    env.python.conda_dependencies.add_pip_package(package)\n",
+    "\n",
+    "inf_config = InferenceConfig(\n",
+    "    environment=env, source_directory=\"./source_dir\", entry_script=\"./score.py\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08c0c387",
+   "metadata": {},
+   "source": [
+    "## Deploy again and call your service"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd084966",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "service = Model.deploy(ws, \"myservice\", [model], inference_config, deployment_config)\n",
```

Comment:
This line should use inf_config

Reply:
renamed one as dummy_inference_config(for deploy on local) and the other ones(for deploy again and redeploy to cloud) as inference_config.

Reply:
renamed one under deploy on local as "dummy_inference_config" and the other ones(for deploy again on local and re-deploy on cloud) as inference_config.